hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/InfoSubcommand.java

```diff
@@ -52,6 +54,11 @@
   @Spec
   private CommandSpec spec;
 
+  @CommandLine.Option(names = { "--json" },
+      defaultValue = "false",
+      description = "Format output as JSON")
+  private boolean json;
+
```

Comment:
Let me stop here for a moment and celebrate this flag. I really like it. I think it should be the standard in all the debug output: we should use either json or multi-line one-json per-line everywhere.

It's not just for the robot test but for everyday usage, jq is very powerful.

:heart:  :heart: